### ISSUE TITLE
Determining Condo BBL When 2 Condos are On One Lot

### ISSUE THREAD

**Issue by mattyschell** [NYCPlanning/db-pluto#227]
Colleagues! 👍🏽 

When more than one billing_bbl exists on a single piece of tax lot land, how do you determine which BBL to output in MAPPLUTO?

For example, from DOFs CONDO dataset, how do you choose which BILLING_BBL to include in MAPPLUTO, 1000937501 or 1000937502?  This is the reverse of the standard condo case, where a single billing_bbl corresponds to several tax lots.

CONDO_NAME                    |CONDO_BASE_BBL|CONDO_BILLING_BBL|
------------------------------|--------------|-----------------|
156 WILLIAM STREET CONDOMINIUM|1000930020    |1000937501       |
156 WILLIAM S1 CONDOMINIUM    |1000930020    |1000937502       |

I am attempting to syncronize DoITT's building "mappluto_bbl" so that we can consistently join buildings to mappluto.  Thanks!
https://github.com/mattyschell/geodatabase-buildings/issues/1#issuecomment-786741931

**Comment by SPTKL** [comment]
@mattyschell you have our attention, we are working on this and will get back to you soon!

**Comment by mattyschell** [comment]
Colleagues!  👍🏽

Though I remain interested in knowing more about how DCP determines a condo_billing_bbl in these cases, we've decided that we can meet our requirements without knowing the answer to this question.

Over at DoITT we will maintain a mappluto_bbl on building footprints.  Whatever borough-block-lots appear in DCP's pluto dataset are the true mappluto_bbls, case closed, no further questions.

With each of your reliable, regular releases of pluto we'll pull those BBLs into our database, joining to DOF's condo dataset, and perform regular updates on building footprints mappluto_bbls with what we've got.  

https://github.com/mattyschell/geodatabase-buildings-condoetl

Thanks!
